test/friends.js

```diff
@@ -92,6 +90,5 @@ tape('createFriendStream - meta', (t) => {
 })
 
 tape('cleanup', (t) => {
-  ssbServer.close()
-  t.end()
+  ssbServer.close(t.end)
```

Comment:
this must have been potentially buggy before?

Reply:
Yeah it was